### ISSUE TITLE
Mask Probing

### ISSUE THREAD

**Issue by PardoAlejo** [NVlabs/GroupViT#34]
Hi guys,

Do you have any plans on releasing the code for evaluating the mask quality (mask probing) mentioned in appendix C.2. table C.2.??
It would be really helpful. Thank you!.

Regards

**Comment by xvjiarui** [comment]
Hi @PardoAlejo 

That part is not cleaned. But I could provide you some part of the code 
```python
jac = 0
for o in objects:
    masko = mask == o
    intersection = masko * th_attn
    intersection = torch.sum(torch.sum(intersection, dim=-1), dim=-1)
    union = (masko + th_attn) > 0
    union = torch.sum(torch.sum(union, dim=-1), dim=-1)
    jaco = intersection / union
    jac += max(jaco)
jac /= len(objects)
jacs += jac
```

It's the same as DINO.

**Comment by PardoAlejo** [comment]
Thank you @xvjiarui
